src/api/users/getTodaysBirthdays.ts

```diff
@@ -0,0 +1,12 @@
+import { useEffect, useState } from 'react';
+import { useAPI } from '@/api/api';
+import { User } from '@/api/users/user.interface';
+
+export default function useTodaysBirthdays(): User[] | null {
+  const [users, setUsers] = useState<User[] | null>(null);
+  const api = useAPI();
+  useEffect(() => {
+    api.get<User[]>('/users/birthdays/today').on('success', (body) => setUsers(body));
```

Comment:
En vrai...
```suggestion
    api.get<User[]>('/users/birthdays/today').on('success', setUsers);
```
Jme suis permis d'ouvrir la PR sur l'api : https://github.com/ungdev/etu-utt-api/pull/24